isaric/parsers/sprintsari-australia.toml

```diff
@@ -0,0 +1,832 @@
+  description = "Was the patient admitted into ICU?"
+  ref = "Y/N"
+
+[subject.has_died]
+  field = "dsterm"
+  values = { 6 = true }
+
+[subject.date_death]
+  field = "death_dt"
+  source_date = "Y-%m-%d %H:%M"
+  description = "Date and time of death"
+
+[visit]
+  country_iso3 = "AUS"
+
+  [visit.subject_id]
+    field = "subjid"
+    sensitive = true
+
+  [visit.visit_id]
+    field = "subjid"
+    sensitive = true
+
+  [visit.icu_admission]
+    field = "hostdat_icu_admit"
+    description = "Was the patient admitted into ICU?"
+    ref = "Y/N"
+
+  [visit.icu_admission_dates]
+    combinedType = "set"
+    excludeWhen = "none"
+
+    [[visit.icu_admission_dates.fields]]
+      field = "hostdat_icu"
+      source_date = "%Y-%m-%d %H:%M"
+      description = "Admission date and time at ICU"
+
+[visit.pathogen_test_date]
+  field = "mbdat_cr"
+  description = "Collection date and time - coronavirus test 1"
+  source_date = "%Y-%m-%d %H:%M"
+  if = { corona_mbcaty = 1 }
+
+[visit.start_date]
+  ref = "admissionDateHierarchy"
+
+[visit.treatment_antibiotics]
+  field = "antibiotic_cmyn"
+  description = "Antibiotic ?"
+  ref = "Y/N"
+
+[visit.treatment_antivirals]
+  field = "antiviral_cmyn"
+  description = "Antiviral agent?"
+  ref = "Y/N"
+
+[visit.treatment_antiviral_type]
+  combinedType = "set"
+  excludeWhen = "none"
+  fields = [
+    { field = "antiviral_cmtrt", description = "IF YES to antiviral agent", values = { 1 = "Ribavirin", 2 = "Lopinavir/Ritonvir", 3 = "Interferon alpha", 4 = "Interferon beta", 5 = "Neuraminidase inhibitor", 6 = "Other" } },
+  ]
+
+[visit.treatment_corticosteroid]
+  field = "corticost_cmyn"
+  description = "Corticosteroid ?"
+  ref = "Y/N"
+
+[visit.treatment_dialysis]
+  field = "daily_rrt_cmtrt"
+  description = "Dialysis / Hemofiltration?"
+  ref = "Y/N"
+
+[visit.treatment_ecmo]
+  field = "daily_ecmo_prtrt"
+  description = "ECMO?"
+  ref = "Y/N"
+
+[visit.treatment_high_flow_nasal_cannula]
+  field = "daily_nasaloxy_cmtrt"
+  description = "High-flow nasal canula oxygen therapy?"
+  ref = "Y/N"
+
+[visit.treatment_inhaled_nitric_oxide]
+  field = "daily_nitritc_cmtrt"
+  description = "Inhaled Nitric Oxide ?"
+  ref = "Y/N"
+
+[visit.treatment_inotropes_vasopressors]
+  field = "daily_inotrope_cmyn"
+  description = "Any vasopressor / inotropic support?"
+  ref = "Y/N"
+
+[visit.treatment_invasive_ventilation]
+  field = "daily_invasive_prtrt"
+  description = "Invasive ventilation?"
+  ref = "Y/N"
+
+[visit.treatment_noninvasive_ventilation]
+  field = "daily_noninvasive_prtrt"
+  description = "Non-invasive ventilation (e.g. BIPAP, CPAP)?"
+  ref = "Y/N"
+
+[visit.treatment_neuromuscular_blocking_agents]
+  field = "daily_neuro_cmtrt"
+  description = "Neuromuscular blocking agents?"
+  ref = "Y/N"
+
+  # [visit.treatment_oxygen_interface] # CHECK: I think interface is just a description of how the oxygen was delivered, which suggests just an observation context rather than a visit attribute.
```

Comment:
Oxygen therapy is a treatment, so the visit table is the appropriate place for it. I'll change the field to type "set" so it can be used to record all the different types of interface (HFNC, nasal cannula, face mask etc) used during a visit.